addons/portal/views/portal_templates.xml

```diff
@@ -81,7 +81,7 @@
             <div t-attf-class="js_language_selector #{_div_classes} d-print-none" t-if="language_selector_visible">
                 <button t-attf-class="btn border-0 dropdown-toggle #{_btn_class or 'btn-sm btn-outline-secondary'}" type="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                     <span t-if="not no_text"
-                            t-attf-class="align-middle #{_txt_class}"
+                            t-attf-class="#{_txt_class}"
```

Comment:
Stable fix: we don't remove classes unless it is really important. In this case, I can only see the difference when the browser is zoomed. At least with the default font, I guess it can be different with other fonts?

Note that with or without this fix, it still feels misaligned to me but it seems to be an optical illusion (maybe due to the fact I would prefer the arrow of the dropdown to be a bit more towards the bottom?) :grimacing: 

So:
- Since stable, use CSS rules or *add* classes (you can add a comment explaining it will be fixed properly in master)
- What cases does it actually fix? (add that in commit message)
- _Feels_ misaligned to you too or is it just me?

Reply:
Hey @qsm-odoo 👋 Thanks for your review :)

> Since stable, use CSS rules or add classes (you can add a comment explaining it will be fixed properly in master)

Ok, so I guess I should directly change this PR to `[IMP]` targeting master, right ? 🤔 

> Note that with or without this fix, it still feels misaligned to me but it seems to be an optical illusion.
> Feels misaligned to you too or is it just me?

Looking at the zoomed-in screenshot in the PR description, the buttons are well aligned, so I guess it's an optical illusion 🤔 . For me, it looks fine 🙂 

Hey @pierre-pde 👋 , what do you think about this ? 

> • What cases does it actually fix? (add that in commit message)

This fixes the alignment for some fonts, such as the system font used in macOS and `Open Sans`, while other fonts already work well (with or without this fix), such as `Roboto` or `Arvo`. (I'll add it in the commit message 👍 )